<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TagReportWidget</class>
 <widget class="QWidget" name="TagReportWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report by tag</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="ReportParamsFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="2">
       <widget class="TagSelector" name="ReportTagEdit" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="DateRangeSelector" name="ReportRange">
        <property name="ItemsList" stdset="0">
         <string notr="true">QTD;YTD;this_year;last_year</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="CurrencyComboBox" name="TotalCurrencyCombo"/>
      </item>
      <item row="0" column="5">
       <spacer name="ReportFrameSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="ReportTagLbl">
        <property name="text">
         <string>Tag:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="TotalCurrencyLbl">
        <property name="text">
         <string>Common currency:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="TableViewWithFooter" name="ReportTableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>4</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::DotLine</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
     </widget>
     <widget class="JalOperationsTabs" name="OperationDetails" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateRangeSelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/custom/date_range_selector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TableViewWithFooter</class>
   <extends>QTableView</extends>
   <header>jal/widgets/custom/tableview_with_footer.h</header>
  </customwidget>
  <customwidget>
   <class>CurrencyComboBox</class>
   <extends>QComboBox</extends>
   <header>jal/widgets/account_select.h</header>
  </customwidget>
  <customwidget>
   <class>JalOperationsTabs</class>
   <extends>QWidget</extends>
   <header>jal/widgets/operations_tabs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagSelector</class>
   <extends>QWidget</extends>
   <header>jal/widgets/reference_selector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
